lib/wings/active_fedora_converter.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+# rubocop:disable Metrics/ClassLength
```

Comment:
this can probably be dropped in a rebase. i added it at the class level in a PR earlier today.